tests/components/technove/snapshots/test_sensor.ambr

```diff
@@ -47,21 +47,6 @@
     'state': '23.75',
   })
 # ---
-# name: test_sensors[sensor.technove_station_current]
```

Comment:
Why does all these state snapshots disappear?

Reply:
That sensor got removed, but syrupy doesn't actively remove removed snapshots (I think it's configured this way in HA)

Reply:
Ah. It wasn't removed but now I see it's all duplicated as the name changed.